From befce32ae86edc14ee43d9756c18c00162feffc0 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Thu, 30 Jul 2009 17:52:48 +0200
Subject: [PATCH 07/19] add a Sony WALKMAN NWZ-S638F as a USB music player

Applied a fixed/slightly adopted version of a patch from
Bernd Schlapsi <brot@gmx.info> (fd.o#22712) to add a
Sony WALKMAN NWZ-S638F as a USB music player.
---
 .../10freedesktop/10-usb-music-players.fdi         |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index 542eaa7..55495f4 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -372,6 +372,12 @@
             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
             <append key="portable_audio_player.audio_folders" type="strlist">music/</append>
           </match>
+	  <!-- Sony Walkman NWZ-S638F -->
+	  <match key="@storage.originating_device:usb.product_id" int="0x38e">
+	    <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+	    <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+	    <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
+	  </match>
 	</match>
 
 	<!-- Deactived entry. Looks as if this is a generic USB chip, since it matches
-- 
1.6.4.4

